splunk-quarkus/build_deploy.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Adapted from: https://github.com/RedHatInsights/drift-backend/blob/master/build_deploy.sh
+
+set -exv
+
+IMAGE_NAME="quay.io/cloudservices/eventing-integrations-splunk"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+if test -f /etc/redhat-release && grep -q -i "release 7" /etc/redhat-release; then
+    # on RHEL7, use docker
+    DOCKER_CONF="$PWD/.docker"
+    mkdir -p "$DOCKER_CONF"
+    docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+    docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
+    docker --config="$DOCKER_CONF" build -t "${IMAGE_NAME}:${IMAGE_TAG}" .
+    docker --config="$DOCKER_CONF" push "${IMAGE_NAME}:${IMAGE_TAG}"
+    for TAG in "latest" "qa"; do
+        docker --config="$DOCKER_CONF" tag "${IMAGE_NAME}:${IMAGE_TAG}" "${IMAGE_NAME}:$TAG"
+        docker --config="$DOCKER_CONF" push "${IMAGE_NAME}:$TAG"
+    done
+else
+    # on RHEL8 or anything else, use podman
+    AUTH_CONF_DIR="$(pwd)/.podman"
+    mkdir -p $AUTH_CONF_DIR
+    export REGISTRY_AUTH_FILE="$AUTH_CONF_DIR/auth.json"
+    podman login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+    podman login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
```

Comment:
It's not great to pass password within args, as the process exec can be intercepted.
The proper way is to use `--password-stdin` option. Check out [this example](https://github.com/RedHatInsights/compliance-backend/blob/8fafcdaa3606fa05e28bba8a97f972d9be03ff9f/deploy/build-deploy-common.sh#L42).

I'll try to change this in their default templates/documentation.